app/views/pages/index.html.erb

```diff
@@ -23,6 +23,9 @@
     <% end %>
     <div class="govuk-button-group">
       <%= govuk_button_link_to t("pages.index.add_question"), type_of_answer_new_path(@form), class:"govuk-!-margin-bottom-3 govuk-!-margin-top-3" %>
+      <% if policy(@form).can_add_page_routing_conditions? %>
+        <%= govuk_button_link_to "Add a question route", routing_page_path(@form), secondary: true, class:"govuk-!-margin-bottom-3 govuk-!-margin-top-3" %>
```

Comment:
This button text should be a translation, but that's a) my fault and b) something I'm happy to deal with in a separate PR (probably the one where we add the rules for this button appearing)

Reply:
Added this to the ACs on the button card